DB_Schema/leaderboard.md

```diff
@@ -55,5 +114,47 @@ Note: make sure to use `__` double underscore
 ## To Create Tables Run a SQL file like
 
 ```
-mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
+$ mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
```

Comment:
I think this command would overwrite the SQL file rather than read from  it.

Reply:
This one is actually if someone wants to create a new database and tables in their own toolsforge

Reply:
@nicolas-raoul mybad I did not notice the arrow